apps/web-giddh/src/app/material-ledger/components/ledger-aside-pane/component/ledger-aside-pane-account/ledger-aside.pane.account.component.scss

```diff
@@ -0,0 +1,107 @@
+@import "../../../../../../assets/css/color.scss";
+:host {
+    position: fixed;
+    left: auto;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    max-width: 760px;
+    width: 100%;
+    z-index: 99999;
+}
+.aside-pane {
+    padding: 0 !important;
+}
+
+:host #close {
+    display: block;
+    position: absolute;
+    right: 0;
+    top: 16px;
+    z-index: 5;
+    border: 0;
+    border-radius: 0;
+    background: transparent;
+    color: $white;
+}
+
+:host .container-fluid {
+    padding-left: 0;
+    padding-right: 0;
+}
+
+:host .aside-pane {
+    max-width: 760px;
+    width: 100%;
+}
+.aside-body {
+    padding: 0 15px;
+}
+.aside-pane-form {
+    background-color: $white;
+    box-shadow: 0px 3px 6px $black-russian;
+    padding: 15px 15px;
+    height: calc(100vh - 90px);
+    overflow: auto;
+    scroll-behavior: smooth;
+    position: relative;
+    margin-top: 15px;
+}
+
+@media (max-width: 991px) {
+    :host {
+        max-width: 500px;
+    }
+    :host .aside-pane {
+        max-width: 500px;
+    }
+}
+
+@media (max-width: 575px) {
+    :host {
+        max-width: 370px;
+    }
+    :host .aside-pane {
+        max-width: 370px;
+    }
+}
+@media (max-width: 375px) {
+    :host {
+        max-width: 310px;
+    }
+    :host .aside-pane {
+        max-width: 310px;
+    }
+}
+@media (max-width: 320px) {
+    :host {
+        max-width: 280px;
+    }
+    :host .aside-pane {
+        max-width: 280px;
+    }
+}
+
+.flexy-child {
```

Comment:
please add this css above media queries

Reply:
This is existing css, i didn't add it. Still i moved it above